templates/service.yaml

```diff
@@ -24,6 +24,6 @@ spec:
       nodePort: {{ .Values.service.nodePort }}
       {{- end }}
       targetPort: {{ .Values.tfe.privateHttpsPort }}
-      appProtocol: https
+      appProtocol: tcp
```

Comment:
I think this is fine to do. If we want to be extra careful and allow consumers to make changes here I think we should lock this in as a entry in values.yaml, with a default of `tcp`, and reference that here.

Consumers can use override values to force it back to https if necessary in their infrastructure.

Reply:
Great suggestion, thank you John.